apps/auth/db/commands.py

```diff
@@ -27,13 +27,12 @@ class CreateUserCommand(superdesk.Command):
         superdesk.Option('--username', '-u', dest='username', required=True),
         superdesk.Option('--password', '-p', dest='password', required=True),
         superdesk.Option('--email', '-e', dest='email', required=True),
-        superdesk.Option('--admin', '-a', dest='admin', required=False),
+        superdesk.Option('--admin', '-a', dest='admin', required=False, default=False),
```

Comment:
yep it does and it does some detection, but when you set it explicitly it doesn't do it.
I've tested it - if you run it with just `--admin` you get `manage.py users:create: error: argument --admin/-a: expected one argument`